index.js

```diff
@@ -5,99 +5,47 @@
 
 'use strict';
 
-var async = require('async'),
-	cheerio = require('cheerio'),
-	request = require('request'),
-	microdata = require('microdata-node');
+var BBPromise = require('bluebird');
+var cheerio = require('cheerio');
+var preq = require('preq');
+var microdata = require('microdata-node');
 
 // Default exported function
-exports = module.exports = function(urlOrOpts, callback){
-	request(urlOrOpts, function(error, response, html){
-		var chtml = cheerio.load(html);
-		exports.parseAll(chtml, function(err, results){
-			callback(err, results);
-		});
+exports = module.exports = function(urlOrOpts) {
+	return preq.get(urlOrOpts
```

Comment:
Yeah, we don't actually use this function upstream in citoid- it's more of a convenience method to demo the library.